### PR TITLE
OCLOMRS-232: The preferred language should not appear in other languages

### PR DESCRIPTION
**Pull request by emmabaye**
# JIRA TICKET NAME:
[The preferred language should not appear in other languages](https://issues.openmrs.org/browse/OCLOMRS-232)

# Summary:
When a user selects the preferred language, it should not appear in other languages selection.

### CODE REVIEW

File: src/components/dashboard/components/dictionary/common/DictionaryModal.jsx [hunk 1]
```diff
@@ -112,9 +112,10 @@ export class DictionaryModal extends React.Component {
        supported_locales,
      },
    } = this.props;
-   if (supported_locales === undefined) {
+   if (supported_locales === undefined || supported_locales === null) {
      return null;
    }
+   console.log("SUPPORTED LOCALES ", supported_locales);
```
Comment (alexmochu) [inline src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: @emmabaye I don't think this console.log is relevant.

Comment (emmabaye) [reply src/components/dashboard/components/dictionary/common/DictionaryModal.jsx]: Thanks for the feedback, @alexmochu. I have removed that line of code.

**Comment by emmabaye** [comment]
> @emmabaye why has this taken 8 days without closing?
I was trying to get the team to review this PR.

**Comment by dkayiwa** [comment]
@emmabaye can you always ping me when a pull request gets into the second day without being reviewed by everyone?

**Comment by emmabaye** [comment]
> @emmabaye can you always ping me when a pull request gets into the second day without being reviewed by everyone?

Yes, I can.

**Pull request merged**